src/index.html

```diff
@@ -17,6 +17,10 @@
     />
```

Comment:
It seems like you're trying to link a SCSS file directly into your HTML. Browsers don't understand SCSS files, they only understand CSS files. You need to compile your SCSS to CSS and link the compiled CSS file here.